utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -217,6 +217,22 @@ def should_build(self, host_target):
         # LLVM will always be built in part
         return True
 
+    @classmethod
+    def compute_cmake_args_for_runtimes(cls, host_target):
+        # Swift expects the old layout for the runtime directory
+        args_for_runtime = '-DLLVM_ENABLE_PER_TARGET_RUNTIME_DIR:BOOL=OFF'
```

Comment:
Note that Windows has started migrating to the new layout. clang already prefers the new layout. We should file an issue to bring the other hosts in line.

Reply:
Created https://github.com/swiftlang/swift/issues/80180